### ISSUE TITLE
Bug report: Hover effect not working on markdown Tables for /docs in light mode

### ISSUE THREAD

**Issue by corwintines** [ethereum/ethereum-org-website#10906]
### Describe the bug

The new design system tables have a hover effect when going over the rows, but this functionality is not working in the markdown pages for the developer docs in light mode since the background color is the same as the hover effect color.

### To reproduce

1. Go to https://ethereumorgwebsitestaging.gatsbyjs.io/en/developers/docs/nodes-and-clients/
2. Scroll down to Table
3. See error where there is no hover effect in light mode

### Expected behavior

Should have a hover effect unless this is intentional

### Screenshots

https://github.com/ethereum/ethereum-org-website/assets/15589226/72712370-8d64-4fcd-a92e-276d3b3edc6a

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

**Comment by justynleung** [comment]
Hello, can I work on this issue?

**Comment by corwintines** [comment]
@justynleung need confirmation from design on what needs to be done first, but after I can assign to you.

**Comment by justynleung** [comment]
@corwintines I am also working on Accordion #10748 issue and finished a prototype (attached a gif at the comment). Sorry that I did not realize someone has to assign the task. Can I also work on that one too?

**Comment by nloureiro** [comment]
@corwintines, this is a good catch.

From the design side, this is not a bug, the hover effect is there, but it's on the same color as the background
![Screen Shot 2023-08-15 09 36 35 AM](https://github.com/ethereum/ethereum-org-website/assets/1120748/a497f368-9bab-467f-a66e-359e5e9345ac)


This migration issue will be resolved when we implement the Design system MD template that proposes the background color to be the same on all pages. Then the hover on tables will be visible.

screenshot from Figma
![Screen Shot 2023-08-15 09 34 45 AM](https://github.com/ethereum/ethereum-org-website/assets/1120748/11518f29-90cc-4cef-96cf-38553abec333)

My take: don´t change anything here, and let's wait for a final integration of the DS to code to fix this.

(@justynleung, sorry, I saw you wanted to fix this, but I'm sure there are plenty of other issues that you might be able to help)

**Comment by wackerow** [comment]
Closing in favor of above solution via DS.
